packages/icons/core/src/index.ts

```diff
@@ -9,29 +9,22 @@ interface IconPath {
   path: string;
 }
 
-function getIconName(path: IconPath['path']): string {
+function getIconFileName(path: IconPath['path']): string {
   // 1. Get the filename (e.g., 'home.svg')
   const filename = path.split('/').pop()!;
-  // 2. Remove the extension (e.g., 'home')
+  return filename;
+}
+
+function getIconName(filename: string): string {
+  // 1. Remove the extension (e.g., 'home')
   const nameWithoutExt = filename.replace(/\.svg$/, '');
-  // 3. Capitalize the first letter for a clean export name (e.g., 'Home')
+  // 2. Capitalize the first letter for a clean export name (e.g., 'Home')
   return nameWithoutExt.charAt(0).toUpperCase() + nameWithoutExt.slice(1);
 }
 
-const iconNames = Object.keys(allIcons).map(getIconName);
-
-const iconPaths = Object.keys(allIcons).reduce(
-  (acc: Record<string, string>, path) => {
-    const name = getIconName(path);
-    // The path starts with '../icons/', but in the published package,
-    // the 'dist' directory is the root of the install, so the icons
-    // are accessed via the 'icons/' folder itself.
-    const relativePath = path.replace('../', '');
-    acc[name] = relativePath;
-    return acc;
-  },
-  {},
-);
+const iconFiles = Object.keys(allIcons).map(getIconFileName);
+const iconNames = Object.keys(iconFiles).map(getIconName);
 
-export const Icons = iconPaths;
+export const IconFiles = iconFiles;
 export const IconNames = iconNames;
+export default iconFiles;
```

Comment:
Not sure I understand the intent of these exports?

<img width="675" height="651" alt="Screenshot 2025-11-03 at 3 43 45 PM" src="https://github.com/user-attachments/assets/72e7bd13-0a5c-4b81-997d-f3150e08bbe8" />

Reply:
It simply made it easy to programmatically display all. Although I'm not sure why you're seeing the names be just the indexes. It was showing the friendly names for me.